### PR TITLE
APP-3334 - Adding text-ellipsis

### PR DESCRIPTION
**Pull request by axeleriksson147**
https://perzoinc.atlassian.net/browse/APP-3334

Text Ellipsis can either be one row or you want it to ellipse after multiple rows.

style="-webkit-line-clamp: {number}" will be added dynamically by the ui-toolkit-components library.

### CODE REVIEW

File: src/components/atoms/_text-ellipsis.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.tk-text-ellipsis {
```
Comment (symphony-adnane) [inline src/components/atoms/_text-ellipsis.scss:1]: hey Axel i'd move it to container components as it has the role of wrapping. 
atoms are unique independent components ;)

File: src/components/atoms/_text-ellipsis.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    -webkit-box-orient: vertical;
+
+    &__multiple-rows {
```
Comment (symphony-adnane) [inline src/components/atoms/_text-ellipsis.scss:7]: From the example I see it;s a variant and not a subcomponent I'm I right?
-- = variant
__ = sub component

Comment (axeleriksson147) [reply src/components/atoms/_text-ellipsis.scss]: Fixed!

File: src/components/atoms/_text-ellipsis.scss (hunk 1 above)
Comment (axeleriksson147) [reply src/components/atoms/_text-ellipsis.scss]: Agreed, fixed!

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**